openapi-bal-service/src/main/java/io/ballerina/openapi/converter/model/OpenAPIInfo.java

```diff
@@ -0,0 +1,85 @@
+/*
+ *  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package io.ballerina.openapi.converter.model;
+
+import java.util.Locale;
+import java.util.Optional;
+
+import static io.ballerina.openapi.converter.Constants.SPLIT_PATTERN;
+
+/**
+ * This {@code OpenAPIInfo} contains details related to openAPI info section.
+ *
+ * @since 2.0.0
+ */
+public class OpenAPIInfo {
+    private String title = null;
+    private String version = null;
+    private String contractPath = null;
+
+    public OpenAPIInfo() {
+    }
+
+    public Optional<String> getTitle() {
+        return Optional.ofNullable(normalizedTitle(this.title));
+    }
+
+    public void setTitle(String title) {
+        this.title = title;
+    }
+
+    public Optional<String> getVersion() {
+        return Optional.ofNullable(this.version);
+    }
+
+    public void setVersion(String version) {
+        this.version = version;
+    }
+
+    // Generate Title
+    private static String normalizedTitle(String serviceName) {
+        if (serviceName != null) {
+            String[] splits = (serviceName.replaceFirst("/", "")).split(SPLIT_PATTERN);
```

Comment:
Let's avoid defining a common constant with name `SPLIT_PATTERN`, as we could use multiple charactors/regex to split in multiple places, which may cause confusion. Choosing a name like `SERVICE_PATH_SEPARATOR` would make more sense. 

Also, isn't this split pattern OS-dependant?

Reply:
yeah, this split pattern includes all the special characters to split and make the title, hope this won't issue for OS. Shall I change SPLIT_PATTERN into REGEX_PATTERN since the same pattern has been used to escape the special characters as well? 🙂  WDYT?

Reply:
Ah I see. In that case maybe we can use something like `SPECIAL_CHAR_REGEX` since the constant `REGEX_PATTERN` does not indicate its intended usage.

Reply:
Yeah sure I will use `SPECIAL_CHAR_REGEX` for it :)